plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/business-verification/style.scss

```diff
@@ -0,0 +1,213 @@
+@import "@wordpress/base-styles/_colors.native.scss";
+
+$gutenberg-blueberry: #3858e9;
+$gutenberg-blueberry-focus: #1d35b4;
+
+.settings-payments-onboarding-modal__step#business_verification {
+	display: flex;
+	flex-direction: column;
+	flex: 1;
+	overflow: hidden;
+}
+
+.settings-payments-onboarding-modal__step-business-verification {
```

Comment:
I think these two selectors look the same:

`.settings-payments-onboarding-modal__step#business_verification`
`.settings-payments-onboarding-modal__step-business-verification`

We should find a better way to differentiate between the two 😅

Reply:
What is the difference between the two? (as in, where would they both be used?) 

I guess one way of doing it is trying to have a hierarchy of gradually more and more specific selectors.

Reply:
I agree that could be confusing, the html structure is as follows:
```
<div id=""business_verificaiton" class="settings-payments-onboarding-modal__step">
   <div class="settings-payments-onboarding-modal__step-business-verification">
   </div>
</div>
```

I plan to change the first css selector to `#business_verification.settings-payments-onboarding-modal__step` which should sort the confusion. WDYT?

Reply:
Yeah, I think that reads easier!

Reply:
Updated with ddb547f(https://github.com/woocommerce/woocommerce/pull/56588/commits/ddb547f51273e97f38f207685ef2600527831f0e)